### ISSUE TITLE
Feature Request: add option to enable saving testmondata on KeyboardInterrupt

### ISSUE THREAD

**Issue by zettabyte** [tarpas/pytest-testmon#32]
Sometimes when building the initial `.testmondata` for a large project, I may need to stop the process but would like all data for tests run so far to be saved so that they at least don't need to be run again to build `.testmondata`.

If possible, it's be nice to have a command-line switch to enable this behavior so when I have to `CTRL-C` the test run and re-start it again later, I won't be starting over from scratch since building the initial `.testmondata` can be very slow.

**Comment by tarpas** [comment]
Thanks, for writing it down. If anybody wants to implement it contact, me. I think a change of storage format would need to be done first.
